mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -121,8 +152,17 @@ function MentoringAssessmentView(runtime, element, mentoring) {
 
         if (isDone())
             renderGrade();
```

Comment:
This code was not added in this PR, but since you did a lot of cleanup, would you mind wrapping the if branch in curly braces?